src/main/java/org/fogbowcloud/manager/core/models/orders/UserData.java

```diff
@@ -1,12 +1,15 @@
 package org.fogbowcloud.manager.core.models.orders;
 
 import org.apache.commons.codec.binary.Base64;
+import org.fogbowcloud.manager.core.models.token.Token;
 
 public class UserData {
 
     private Long id;
     private String content;
 
+    public UserData() {}
```

Comment:
is this empty constructor necessary by reflection stuff?

Reply:
... can we remove it?

Reply:
Same as comment above. It is how Spring Boot instantiate a object parameterized in body request. There is a UserData field in ComputeOrder.